pdfly/cli.py

```diff
@@ -37,6 +38,29 @@ def version_callback(value: bool) -> None:
     ),
     rich_markup_mode="rich",  # Allows to pretty-print commands documentation
 )
+@entry_point.command(name="sign", help="Sign a PDF file using a PKCS12 (.p12) certificate.")
```

Comment:
```suggestion
@entry_point.command(name="sign", help=pdfly.sign.__doc__)
```
As you can see for other subcommands, we usually provide a lengthy description there,
that is stored in the dedicated module top docstring.

Reply:
Previously it was I didn't understand the role of three quotes, sorry, I corrected it.